### ISSUE TITLE
Scripts erroring out with the new Az module

### ISSUE THREAD

**Issue by abraina** [Azure/azure-powershell#7892]
My customer is running into errors while executing cmdlets in the new Az module. He has installed .Net 4.7.2 and we have verified PS version is 5.1. 

Sample Cmdlet (from customer machine) :
-----------------------------------------------------
```
PS E:\Scripts> Get-AzRecoveryServicesBackupProtectionPolicy -Name $backupPolicy
Get-AzRecoveryServicesBackupProtectionPolicy : An error occurred creating the configuration section handler for
appSettings: Type 'System.Configuration.AppSettingsSection' does not inherit from
'System.Configuration.IConfigurationSectionHandler'.
(C:\Windows\Microsoft.NET\Framework64\v4.0.30319\Config\machine.config line 14)
At line:1 char:1
+ Get-AzRecoveryServicesBackupProtectionPolicy -Name $backupPolicy
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Get-AzRecoveryS...rotectionPolicy], ConfigurationErrorsException
    + FullyQualifiedErrorId : System.Configuration.ConfigurationErrorsException,Microsoft.Azure.Commands.RecoveryServi
  ces.Backup.Cmdlets.GetAzureRmRecoveryServicesBackupProtectionPolicy
----------------------------------------------------------
```

OS : Windows 10 (x64)

PS Version :

```
Name                           Value
----                           -----
PSVersion                      5.1.14393.2608
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14393.2608
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```
.Net Version :

```
[11/23/18,08:58:35] Install state for .NET Framework 1.0: not installed.
[11/23/18,08:58:35] Install state for .NET Framework 1.1: not installed.
[11/23/18,08:58:35] Install state for .NET Framework 2.0: not installed.
[11/23/18,08:58:35] Install state for .NET Framework 3.0: not installed.
[11/23/18,08:58:35] Install state for .NET Framework 3.5: not installed.
[11/23/18,08:58:35] Install state for .NET Framework 4 Client: installed with no service packs.
[11/23/18,08:58:35] Install state for .NET Framework 4 Full: installed with no service packs.
[11/23/18,08:58:35] Install state for .NET Framework 4.5: installed with no service packs.
[11/23/18,08:58:35] Install state for .NET Framework 4.5.1: installed with no service packs.
[11/23/18,08:58:35] Install state for .NET Framework 4.5.2: installed with no service packs.
[11/23/18,08:58:35] Install state for .NET Framework 4.6: installed with no service packs.
[11/23/18,08:58:35] Install state for .NET Framework 4.6.1: installed with no service packs.
[11/23/18,08:58:35] Install state for .NET Framework 4.6.2: installed with no service packs.
[11/23/18,08:58:35] Install state for .NET Framework 4.7: installed with no service packs.
[11/23/18,08:58:35] Install state for .NET Framework 4.7.1: installed with no service packs.
[11/23/18,08:58:35] Install state for .NET Framework 4.7.2: installed with no service packs.
[11/23/18,08:58:45] Starting verification for product .NET Framework 4.7.2
```

I can get the Debug stream if required. He is quite keen on moving to the Az module before AzureRM is deprecated.

**Comment by cormacpayne** [comment]
@dragonfly91 Hey Samuel, would you mind taking a look at this issue?

**Comment by markcowl** [comment]
Here is a stack trace from Resolve-AzError

```c#
PS C:\Users\markcowl> Resolve-AzError


   HistoryId: 14


Message        : An error occurred creating the configuration section handler for appSettings: Type
                 'System.Configuration.AppSettingsSection' does not inherit from
                 'System.Configuration.IConfigurationSectionHandler'.
                 (C:\Windows\Microsoft.NET\Framework64\v4.0.30319\Config\machine.config line 14)
StackTrace     :    at System.Configuration.BaseConfigurationRecord.FindAndEnsureFactoryRecord(String configKey,
                 Boolean& isRootDeclaredHere)
                    at System.Configuration.BaseConfigurationRecord.GetSectionRecursive(String configKey, Boolean
                 getLkg, Boolean checkPermission, Boolean getRuntimeObject, Boolean requestIsHere, Object& result,
                 Object& resultRuntimeObject)
                    at Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.ServiceClientAdapterNS.ServiceClientAda
                 pter.get_ResourceProviderNamespace()
                    at Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.RecoveryServicesBackupCmdletBase.SetupH
                 ttpClientPipeline()
                    at Microsoft.WindowsAzure.Commands.Utilities.Common.AzurePSCmdlet.BeginProcessing()
                    at Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.RecoveryServicesBackupCmdletBase.BeginP
                 rocessing()
                    at System.Management.Automation.Cmdlet.DoBeginProcessing()
                    at System.Management.Automation.CommandProcessorBase.DoBegin()
Exception      : System.Configuration.ConfigurationErrorsException
InvocationInfo : {Get-AzRecoveryServicesBackupProtectionPolicy}
Line           : Get-AzRecoveryServicesBackupProtectionPolicy
Position       : At line:1 char:1
                 + Get-AzRecoveryServicesBackupProtectionPolicy
                 + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
HistoryId      : 14

Message        : Type 'System.Configuration.AppSettingsSection' does not inherit from
                 'System.Configuration.IConfigurationSectionHandler'.
StackTrace     :    at System.Configuration.TypeUtil.VerifyAssignableType(Type baseType, Type type, Boolean
                 throwOnError)
                    at System.Configuration.MgmtConfigurationRecord.CreateSectionFactory(FactoryRecord factoryRecord)
                    at System.Configuration.BaseConfigurationRecord.FindAndEnsureFactoryRecord(String configKey,
                 Boolean& isRootDeclaredHere)
Exception      : System.TypeLoadException
InvocationInfo : {Get-AzRecoveryServicesBackupProtectionPolicy}
Line           : Get-AzRecoveryServicesBackupProtectionPolicy
Position       : At line:1 char:1
                 + Get-AzRecoveryServicesBackupProtectionPolicy
                 + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
HistoryId      : 14
```

**Comment by franzoir** [comment]
We have the same problem. 
`Name                           Value                                                                                                                                                                                
----                           -----                                                                                                                                                                                
PSVersion                      5.1.17134.407                                                                                                                                                                        
PSEdition                      Desktop                                                                                                                                                                              
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                                                                              
BuildVersion                   10.0.17134.407                                                                                                                                                                       
CLRVersion                     4.0.30319.42000                                                                                                                                                                      
WSManStackVersion              3.0                                                                                                                                                                                  
PSRemotingProtocolVersion      2.3                                                                                                                                                                                  
SerializationVersion           1.1.0.1             `

`PS C:\Windows\system32> Get-InstalledModule Az

Version    Name                                Repository           Description                                                                                                                                     
-------    ----                                ----------           -----------                                                                                                                                     
0.6.1      Az                                  PSGallery            Azure Resource Manager Module. Cmdlets to manage resources in Azure.  This module is compatible with PowerShell and PowerShell Core`

Let me know if you need more information.

**Comment by markcowl** [comment]
Fixed in 1.0
